packages/frontend-shared/src/locales/en-US.json

```diff
@@ -598,7 +598,7 @@
       "description": "Complete the steps below to migrate your project to Cypress 10",
       "step1": {
         "title": "Rename existing specs",
-        "description": "In this step, we'll automatically rename and move your existing spec files.",
+        "description": "In this step, we'll automatically change existing spec files to match the new convention.",
```

Comment:
This also seems too broad compared to what we have now. How about:

> In this step, we'll automatically rename and/or move your existing spec files as needed.

Which keeps the scope narrow to what we will actually do, and allows it to be correct even if we are only doing one thing or the other. We could leave all other text the same I think.

Reply:
I actually had some variation of this but ended up flip flopping back to "update", but happy to go with this.